layouts/default.vue

```diff
@@ -56,11 +56,31 @@ export default {
   head() {
     return {
       meta: [
+        {
+          hid: 'og:title',
+          property: 'og:title',
+          content: this.appName
+        },
         {
           hid: 'og:url',
           property: 'og:url',
           content: `${process.env.baseUrl}${this.$route.path}`
         },
+        {
+          hid: 'twitter:card',
+          property: 'twitter:card',
+          content: 'summary'
+        },
+        {
+          hid: 'twitter:site',
+          property: 'twitter:site',
+          content: '@HestiaLabs'
+        },
```

Comment:
We might want to move these two tags to `meta.twitterCard` and `meta.twitterSite` in `nuxt.config.js`